install/install.pl

```diff
@@ -42,7 +42,7 @@
 my $iridaDir = $ENV{HOME}."/.irida/";
 my $confFile = "$iridaDir/ngs-archive-linker.conf";
 
-$ret = option("Install config file to $confFile?","y");
+my $ret = option("Install config file to $confFile?","y");
```

Comment:
This is just to fix a missing variable init in the install script.

Reply:
Sounds good